src/test/java/org/broadinstitute/hellbender/tools/walkers/coverage/DepthOfCoverageIntegrationTest.java

```diff
@@ -0,0 +1,259 @@
+package org.broadinstitute.hellbender.tools.walkers.coverage;
+
+import com.opencsv.CSVReader;
+import org.broadinstitute.hellbender.CommandLineProgramTest;
+import org.broadinstitute.hellbender.testutils.ArgumentsBuilder;
+import org.broadinstitute.hellbender.utils.SimpleInterval;
+import org.broadinstitute.hellbender.utils.io.IOUtils;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+import java.io.File;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.*;
+import java.util.stream.Collectors;
+import java.util.stream.StreamSupport;
+
+public class DepthOfCoverageIntegrationTest extends CommandLineProgramTest {
+
+    private String getDoCExtensionFromFile(File docOutputFile, String basename) {
+        String[] split = docOutputFile.getAbsolutePath().split(basename);
+        if (split.length == 2) {
+            return split[1];
+        } else if (split.length == 1) {
+            return "";
+        }
+        Assert.fail("There was a problem loading DoCExtension: "+basename);
+        return null;
+    }
+
+    private File getExpectedDataDir() {
+        return getTestFile( "expected/");
+    }
+
+    @Test
+    public void testBaseOutputNoFiltering() throws IOException {
+        final String expectedBaseName = "depthofcoveragenofiltering";
+        final File baseOutputFile = createTempDir("depthofcoveragenofiltering");
+        final File output = IOUtils.createTempFileInDirectory( "depthofcoveragenofiltering", ".csv", baseOutputFile);
+
+        String cmd = "-R "+hg38Reference+" " +
+                "-I "+largeFileTestDir + "multiSampleSubsetted.bam " +
+                "-L "+ getTestFile("artificial.target_region.interval_list ")+
+                "--min-base-quality 0 -dels --print-base-counts -pt readgroup -pt sample -pt platform -pt library --output-format CSV -ct 10 -ct 15 -ct 20 -ct 25";
+        cmd += " -O "+output.getAbsolutePath();
+        runCommandLine(cmd.split(" "));
+
+        File[] actualFiles = baseOutputFile.listFiles();
+
+        compareOutputDirectories(expectedBaseName, output.getName(), actualFiles);
+    }
+
+    @Test
+    // Asserting that the per-base statistics are reasonable when run without an interval list file
+    public void testCoverageBehaviorWhenProvidedNoIntervalFile() throws IOException {
+        final String expectedBaseName = "testCoverageBehaviorWhenProvidedNoIntervalFile";
+        final File baseOutputFile = createTempDir("testCoverageBehaviorWhenProvidedNoIntervalFile");
+        final File output = IOUtils.createTempFileInDirectory( "testNoCoverageDueToFiltering", ".csv", baseOutputFile);
+
+        String cmd = "-R "+hg38Reference+" " +
+        "-I "+largeFileTestDir+"multiSampleSubsetted.bam"+
+        " -O "+output.getAbsolutePath() + " -L chr1:1570000-1571000 --omit-interval-statistics --omit-locus-table --omit-per-sample-statistics";
```

Comment:
This test case claims to be testing the behavior with no interval file, but it does provide an interval to `-L` (which is equivalent to providing an interval file containing 1 interval). What are you actually trying to test here? The `--omit-interval-statistics`, `--omit-locus-table`, etc. arguments?